### PR TITLE
feat: Added support for nats

### PR DESCRIPTION
**Pull request by drkfmorton**
Signed-off-by: Kyle Morton <kylex.f.morton@intel.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)

### CODE REVIEW

**Review by bnevis-i — APPROVED** [review]
LGTM

**Comment by bnevis-i** [comment]
Soft link to #134 (hit max # of github linkages)

File: deployment/helm/values.yaml [hunk 1]
```diff
@@ -541,3 +571,11 @@ edgex:
     sharedVolumesClassName: "rook-cephfs"
     # If this value is set to "ReadWriteOnce" it will trigger pod affinity rules.
     sharedVolumesAccessMode: "ReadWriteMany"
+
+  features:
+    # Available messagebus options: redis|mqtt|nats
```
Comment (lenny-intel) [inline deployment/helm/values.yaml:576]: # Available messagebus options: redis|mqtt|nats-core|nats-jetstream

File: deployment/helm/values.yaml [hunk 2]
```diff
@@ -98,8 +100,12 @@ edgex:
...
         pullPolicy: IfNotPresent
       httpexport:
-        repository: nexus3.edgexfoundry.org:10004/app-service-configurable
-        tag: "latest"
+        repository: edgexfoundry/app-service-configurable
+        tag: "2.3.0"
+        pullPolicy: IfNotPresent
+      externalmqtttrigger:
+        repository: edgexfoundry/app-service-configurable
+        tag: "2.3.0"
```
Comment (lenny-intel) [inline deployment/helm/values.yaml:108]: If nats is used then the locally built image must be referenced. Same for all the services

**Review by lenny-intel — COMMENTED** [review]
Seems the PR was beyond NATs and is incomplete for supporting NATS

File: deployment/helm/values.yaml (hunk 2 above)
Comment (bnevis-i) [reply deployment/helm/values.yaml:108]: Yes, got another issue for that:

https://github.com/edgexfoundry/edgex-examples/issues/157

**Pull request merged**